src/opm/test_util/EclFilesComparator.cpp

```diff
@@ -405,6 +406,20 @@ void RegressionTest::deviationsForCell(double val1, double val2, const std::stri
 
 
 
+namespace {
+    double getCellVolume(const ecl_grid_type* ecl_grid, const int globalIndex) {
+        std::vector<double> x(8, 0.0);
+        std::vector<double> y(8, 0.0);
+        std::vector<double> z(8, 0.0);
+        for (int i = 0; i < 8; i++) {
+            ecl_grid_get_cell_corner_xyz1(ecl_grid, globalIndex, i, &x.data()[i], &y.data()[i], &z.data()[i]);
```

Comment:
Just out of curiosity: Why do you write `&x.data()[i]` here as opposed to the more typical `&x[i]`?

Reply:
> Why do you write &x.data()[i] here as opposed to the more typical &x[i]?

It was in the commit I used as a base, I agree that it is less clear than the alternative. Will change.